### PR TITLE
SW-2050: Accessions Page Columns

### PR DESCRIPTION
**Pull request by joshbuchea**
This PR removes & renames accession page columns that are no longer valid after the update to v2 for accessions.

### CODE REVIEW

**Comment by joshbuchea** [comment]
Just noticed the cypress tests are failing. Will fix.

**Review by karthikbtf — APPROVED** [review]
Thanks 🙏

File: src/components/seeds/database/EditColumns.tsx [hunk 1]
```diff
@@ -217,10 +210,8 @@ const sections: Section[] = [
         COLUMNS_INDEXED.viabilityTests_seedsSown,
         COLUMNS_INDEXED.viabilityTests_viabilityTestResults_seedsGerminated,
         COLUMNS_INDEXED.cutTestSeedsEmpty,
-        COLUMNS_INDEXED.latestViabilityPercent,
       ],
       [
-        COLUMNS_INDEXED.latestViabilityTestDate,
         COLUMNS_INDEXED.viabilityTests_substrate,
         COLUMNS_INDEXED.cutTestSeedsCompromised,
         COLUMNS_INDEXED.viabilityTests_notes,
```
Comment (constanzauanini) [inline src/components/seeds/database/EditColumns.tsx:217]: I think we could move viabilityTests_percentGerminated to this group (this is just visual, so we don't have an empty space)

File: src/components/seeds/database/EditColumns.tsx [hunk 2]
```diff
@@ -181,18 +181,11 @@ const sections: Section[] = [
   },
   {
     name: 'Processing and Drying',
-    options: [
-      [COLUMNS_INDEXED.totalQuantity, COLUMNS_INDEXED.viabilityTestType, COLUMNS_INDEXED.dryingStartDate],
-      [COLUMNS_INDEXED.dryingEndDate, COLUMNS_INDEXED.targetStorageCondition, COLUMNS_INDEXED.processingNotes],
-    ],
+    options: [[COLUMNS_INDEXED.totalQuantity, COLUMNS_INDEXED.viabilityTestType], [COLUMNS_INDEXED.dryingEndDate]],
```
[the remark this answers is not in the document]
Comment (joshbuchea) [reply src/components/seeds/database/EditColumns.tsx:184]: @constanzauanini you got it, great suggestion 🙌

**Comment by joshbuchea** [comment]
@constanzauanini I implemented the changes you suggested, thanks again 🙌  Could you please review when you have a moment and confirm the changes I made look ok? 🙏 I'm happy to adjust if needed.

**Review by constanzauanini — APPROVED** [review]
Excellent! Thanks for the change @joshbuchea !

**Comment by joshbuchea** [comment]
@constanzauanini you bet 🙌 Teamwork makes the dream work! 😄

**Pull request merged**